package.json

```diff
@@ -6,10 +6,10 @@
   "license": "MIT",
   "repository": "phosphor-icons/phosphor-vue",
   "homepage": "https://phosphoricons.com",
-  "main": "dist/phosphor-vue.ssr.js",
-  "browser": "dist/esm/entry.js",
-  "module": "dist/esm/entry.js",
-  "unpkg": "dist/phosphor-vue.min.js",
```

Comment:
Please keep the `unpkg` entry, and point to the UMD bundle... I think Unpkg falls back to the `main` entry by default, but let's keep it there to be safe.